lib/pact_broker/client/pacts.rb

```diff
@@ -21,6 +21,11 @@ def publish options
         end
       end
 
```

Comment:
OK, I renamed the method and use old `args` syntax.

Also, should I pass consumer name and provider name to `version_published?` instead of pact json? I don't get well that why does `#get` recieve individual names.